src/Tools/dotnet-watch/test/GlobbingAppTests.cs

```diff
@@ -25,6 +26,7 @@ public GlobbingAppTests(ITestOutputHelper logger)
         [InlineData(true)]
         [InlineData(false)]
         [SkipOnHelix] // https://github.com/aspnet/AspNetCore/issues/8267
+        [Flaky("https://github.com/aspnet/AspNetCore/issues/8267", FlakyOn.AzP.Linux, FlakyOn.AzP.macOS)]
```

Comment:
I think we should go ahead and skip these tests rather than marking them as flaky. They are failing ~90% of the time.